README.md

```diff
@@ -223,16 +223,32 @@ Display progress in new lines instead of displaying a busy spinner and the statu
 
 ## :rescue_worker_helmet: Troubleshooting
 
-*  Unable to connect to Contentful through your proxy? Try to set the `rawProxy` option to `true`.
+### Proxy
+
+Unable to connect to Contentful through your proxy? Try to set the `rawProxy` option to `true`.
+
+```javascript
+contentfulExport({
+  proxy: 'https://cat:dog@example.com:1234',
+  rawProxy: true,
+  ...
+})
+```
+
+### Error: 400 - Bad Request - Response size too big.
+
+Lower the amount of entites received within a single request by setting the maxAllowedLimit option:
```

Comment:
@marcolink could we link out do docs, describing what this error means, too?

Thinking of something along the lines "Contentful response sizes are limited (find more info in our technical limit docs). To resolve this issue, limit the queried entries" or something. 🙈

Reply:
@stefanjudis good point - please review the updated version!